index.php

```diff
@@ -0,0 +1,135 @@
+                        <ul>
+                            <li>It is the size of the unpacked project which is counted, not the compressed archive size.</li>
+                        </ul>
+                    </li>
+
+                    <li>It doesn't contain anything that violates any Canadian federal law.</li>
+                    <li>Isn't porn, doesn't have gratuitous nudity and explicit or implict sexual acts.</li>
+                    <li>It is a video game.</li>
+                    <li>You have a full month to do it, so it is expected it will be made during that month (may 2023).
+                        <ul>
+                            <li>Premade art, sound assets engine code are allowed.</li>
+                        </ul>
+                    </li>
+                    <li>It isn't a game mod or hack.</li>
+                    <li>It isn't an online multiplayer-only game.
+                        <ul>
+                            <li>Games with an online multiplayer component are fine as long as the main part of the game is entirely playable in solo. This is for judging reasons.</li>
+                            <li>Excluding file size, Judges are free to not include the multiplayer part of the game in their evaluation.</li>
+                            <li>If there's a dedicated server application, those files also count and must be included in the submission.</li>
+                        </ul>
+                    </li>
+                    <li>It doesn't stream nor download content from the internet.</li>
+                    <li>It is available for free for the duration of the judging period.</li>
+                    <li>It can be downloaded for the duration of the judging period.</li>
+                    <li>For clarification, a desktop application is an application that isn't a web application.
+                        <ul>
+                            <li>Something that embeds a web game into a binary desktop application, like electron or tauri is allowed. However, please keep in mind that some of these tools, like electron for example, will blow up the size of the executable way past the 4mb limit, so alternatives like tauri should be preferred.</li>
+                        </ul>
+                    </li>
+                </ol>
+
+                <a href="#">View Full Rules</a>
+            </section>
+
+            <section>
+                <header>
+                    <h2 id="resources">
+                        <a href="#resources">Resources</a>
+                    </h2>
+                </header>
+
+                <p>Check out the <a href="#">Resources</a> section for guidance on ways of making games that take up as little storage space as possible!</p>
+            </section>
+
+            <section>
+                <header>
+                    <h2 id="community">
+                        <a href="#community">Community</a>
+                    </h2>
+                </header>
```

Comment:
these can probably be automated with php templates. It makes no sense to write by hand.

Reply:
Planned to do this in a separate PR but yeah, I might as well just do it now